writer_complex_test.go

```diff
@@ -500,4 +501,82 @@ func (s *WriterComplexSuite) TestWriteComplexObject() {
 		// 55
 		0x6e, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0,
 	}, buf.Bytes())
+
+	pbuf := &bytes.Buffer{}
+	err := Print(pbuf, bufio.NewReader(buf), FullPackageRecordPyPI{})
+	s.Require().Nil(err)
+	s.Require().Equal(`
+--------------------------------------------------------
+                FullPackageRecordPyPI[1]                
+--------------------------------------------------------
+homepage (string): http://homepage.com
+cname (string): numpy
+pname (string): Numpy
+classifiers (array(2))
```

Comment:
Small nit: arrays and indexed arrays don't output a colon.

Reply:
Fixed!